hadoop-ozone/dist/src/main/smoketest/security/ozone-secure-tenant.robot

```diff
@@ -85,10 +85,6 @@ SetSecret Failure For Invalid Secret 2
     ${rc}  ${output} =  Run And Return Rc And Output  ozone tenant user setsecret 'tenantone$testuser' --secret=short
                         Should contain   ${output}         Secret key length should be at least 8 characters
 
-GetSecret Success
```

Comment:
Why this one is removed?

Reply:
The test intends to test getsecret for tenant. Refer to this comment: https://github.com/apache/ozone/pull/4538#discussion_r1161453347. Theeffect of this change on Tenant getsecret- It makes it obsolete. Hence the test was removed.

Reply:
I have added the corrected test again